src/main/assets/tax_profile.html

```diff
@@ -5,7 +5,7 @@
 <body>
 <div id="abound-ui-wrapper"></div>
 <style>
-      .abound-tax-profile {
+      .abound-w9-collection {
```

Comment:
The CSS class changed. Without this change, the tax document screen looks like this:
![Screenshot_20240326-150211](https://github.com/Kidizen/abound-android-client-sdk/assets/1857059/aea6c5d0-4810-4687-9245-ac88372039bf)

Afer this fix it looks like this:
![Screenshot_20240326-150242](https://github.com/Kidizen/abound-android-client-sdk/assets/1857059/0809e4a1-d7e6-43d3-b145-a4fbe7447e57)